### DISCUSSION TITLE
Anyone know how to bridge revolt to matrix

### DISCUSSION THREAD

**Discussion by FillingTheVoidOnYT** [stoatchat/discussions#437] [Q&A] [answered]
Title

**Comment by Rexogamer** [comment, marked as answer]
We don't currently have a native bridge for Matrix - however, you can connect the Revolt channel to a Discord channel with AutoMod, then connect that to the Matrix room with t2bot (which we do for the Revolt Lounge).

**Reply by FillingTheVoidOnYT** [reply]
is that revolt lounge still being maintained under this link 
https://matrix.to/#/#revolt-chat:clustor.net

**Reply by Rexogamer** [reply]
Yes.
